scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/constraints/activities/ActivityCreationTemplate.java

```diff
@@ -458,21 +474,23 @@ private  Optional<SchedulingActivityDirective> rootFindingHelper(
               durationHalfEndInterval,
               startInterval.start,
               startInterval.end,
-              20);
+              maxIterations);
 
       // TODO: When scheduling is allowed to create activities with anchors, this constructor should pull from an expanded creation template
       final var lastActivityTested = result.history().getLastEvent();
+      logger.info("Finished rootfinding: SUCCESS");
+      history.logHistory();
       return Optional.of(lastActivityTested.get().activity);
     } catch (EquationSolvingAlgorithms.ZeroDerivativeException zeroOrInfiniteDerivativeException) {
-      logger.debug("Rootfinding encountered a zero-derivative");
+      logger.info("Rootfinding encountered a zero-derivative");
     } catch (EquationSolvingAlgorithms.InfiniteDerivativeException infiniteDerivativeException) {
-      logger.debug("Rootfinding encountered an infinite-derivative");
+      logger.info("Rootfinding encountered an infinite-derivative");
     } catch (EquationSolvingAlgorithms.DivergenceException e) {
-      logger.debug("Rootfinding diverged");
+      logger.info("Rootfinding diverged");
     } catch (EquationSolvingAlgorithms.ExceededMaxIterationException e) {
-      logger.debug("Too many iterations");
+      logger.info("Rootfinding exited because too many iterations ("+maxIterations+")");
     } catch (EquationSolvingAlgorithms.NoSolutionException e) {
-      logger.debug("No solution");
+      logger.info("Rootfinding: no solution");
```

Comment:
Minor, can ignore: consistency
```suggestion
      logger.info("Rootfinding found no solution");
```